Dockerfile

```diff
@@ -42,6 +42,7 @@ RUN cd extract-test-code \
 # Pack together things to copy to the runner container
 COPY bin/run.sh bin/run.sh
 COPY bin/smoke_test.sh bin/smoke_test.sh
+COPY bin/run_all_exercises.sh bin/run_all_exercises.sh
```

Comment:
Instead of including the shell file into the docker image (which is fine), an alternative is to map the file into the container at runtime: https://github.com/exercism/generic-test-runner/blob/main/bin/run-tests-in-docker.sh#L28

Reply:
Oh that means I could be running this entirely from the elm repo, that might be a better choice. I will try it out before merging this. 
Thank you!

Reply:
agreed, this matters mainly for the elm repo, as this is the one we wanted to have these tests for. So let's keep it in the main elm repo